src/myr/Myr.js

```diff
@@ -94,6 +98,9 @@ class Myr {
         this.cursor = {
             color: "red",
             texture: "",
+            materialtype: MaterialType.SPECULAR,
```

Comment:
(this applies to wherever cursor.materialtype is mentioned) this should probably be camel case (i.e. "materialType") to be consistent with other variables like "textureColoring"